src/main/java/appeng/container/implementations/ContainerPatternTerm.java

```diff
@@ -450,6 +459,11 @@ public void detectAndSendChanges()
 				this.craftingMode = this.ct.isCraftingRecipe();
 				this.updateOrderOfOutputSlots();
 			}
+
+			if( this.substitute != this.ct.isSubstitution() )
+			{
+				this.substitute = this.ct.isSubstitution();
+			}
```

Comment:
Hm yeah. Mostly pointless. I think I left it intentionally there in case it needed something similar to the crafting mode and `updateOrderOfOutputSlots()`.